.github/workflows/pr-rtd-link.yml

```
@@ -0,0 +1,19 @@
+# The ReadTheDocs preview link is "hidden" within the GitHub "Checks"
+# interface. For users who don't know this, finding the preview link may be
+# very difficult or frustrating. This workflow makes the link more
+# findable by updating PR descriptions to include it.
+name: "Add ReadTheDocs preview link to PR descriptions"
+
+on:
+  pull_request_target:
```

This action recommends specifying 
```suggestion
on:
  pull_request_target:
    types:
       - opened
```
and I believe the link is stable so wouldn't need to trigger on any other event types. Is there a reason you don't include the type?

---

Also, I lean towards only doing this on PRs with docs changes -- is there a good reason to always build the preview?

---

Both good calls!

We will always build the preview no matter what with our current RTD configuration, but there may be a way to tweak that in readthedocs.yml. I just don't know of it :) This new workflow only surfaces the link, so yeah maybe it makes sense to add a `paths` filter too! Even on PRs without docs changes, you can access the RTD preview from the "checks" area of the PR interface.

---

@mfisher87, if RTD generates it no matter what, let's always add the link. It sure can't hurt to have it easy to look at.